### ISSUE TITLE
If I want to add new class to the Yolov3 model with trained coco weight.

### ISSUE THREAD

**Issue by lianuo** [andy-yun/pytorch-0.4-yolov3#17]
@andy-yun Thank you for so much effort in the repo.
I have a question,
If I want to train a model which could detect a class not in the COCO,I have the image and label,could I modify the cfg file of coco ,and transfer the lower layers weight and freeze then ,and then just train the top layers related to the class number?
Thank you ~

**Comment by andy-yun** [comment]
Unfortunately, I don't know exactly the updating method. 
But, If you want to update just 3 yolo layers, you should all flag off by set the required_grad=False.
You should modify the source code by adopting the following discussion;
https://discuss.pytorch.org/t/train-a-network-by-updating-only-part-of-the-weights/2761/3
I recommend you that you should train your network using pretrained weights.

**Comment by lianuo** [comment]
thank you so much，i will try it
